yoonit-camera/src/main/java/ai/cyberlabs/yoonit/camera/analyzers/face/FaceAnalyzer.kt

```diff
@@ -49,7 +49,7 @@ class FaceAnalyzer(
     /**
      * Responsible to manipulate everything related with the face bounding box.
      */
-    private val faceBoundingBoxController = FaceBoundingBoxController(
+    private val faceBoundingBoxController = FaceCoordinatesController(
```

Comment:
The class name changed from `FaceBoundingBoxController` to `FaceCoordinatesController`.
But the var name continue as `faceBoundingBoxController`.